Apple-TV/PlaybackSettings.swift

```diff
@@ -0,0 +1,73 @@
+/*****************************************************************************
+ * VLC for iOS
+ *****************************************************************************
+ * Copyright (c) 2018 VideoLAN. All rights reserved.
+ * $Id$
+ *
+ * Authors: Carola Nitz <caro # videolan.org>
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+import Foundation
+
+@objcMembers
+class PlaybackSettings: NSObject, Codable {
```

Comment:
In this case since it's a data structure, what do you think about using a structure?